openlibrary/i18n/pt/messages.po

```diff
@@ -1980,7 +1985,7 @@ msgid ""
 "Like, \"Agatha Christie\" or \"Jean-Paul Sartre.\" We'll also do a live "
 "check to see if we already know the author."
 msgstr ""
-"Como «Agatha Christie» ou «Jean-Paul Sartre». Também vamos verificar ao vivo "
+"Como \"Agatha Christie\" ou \"Jean-Paul Sartre\". Também vamos verificar ao vivo "
```

Comment:
What is the correct quoting to use for Portuguese?

Reply:
It would be " as it is in English, if I understood the question correctly.